pkg/cli/kubernetes/kubernetes_test.go

```diff
@@ -49,6 +50,19 @@ func TestEnsureNamespace(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestDeleteNamespace(t *testing.T) {
+	namespace := "radius-test"
+	f := k8sfake.NewSimpleClientset(&v1.Namespace{ObjectMeta: meta_v1.ObjectMeta{Name: namespace}})
+
+	ctx := context.TODO()
```

Comment:
Use `context.Background()` instead. https://stackoverflow.com/questions/74239074/context-todo-or-context-background-which-one-should-i-prefer